compiler/rustc_middle/src/mir/interpret/error.rs

```diff
@@ -306,6 +306,9 @@ impl fmt::Display for UndefinedBehaviorInfo<'_> {
                 ptr.alloc_id,
                 allocation_size.bytes()
             ),
+            DanglingIntPointer(_, CheckInAllocMsg::InboundsTest) => {
+                write!(f, "null pointer is not allowed for this operation")
+            }
```

Comment:
This special printing case existed before but at some point turned into dead code which was removed in https://github.com/rust-lang/rust/pull/84903 -- but the code is good, it just should not be dead. :)